src/ui/home/containers/claims.tsx

```diff
@@ -19,6 +23,12 @@ export const ClaimsContainer = (props: ClaimsContainerProps) => {
     props.checkLocalAuthSet()
   }, [])
 
+  useDisableBackButton(() => {
+    // return true (disable back button) if we are focused
+    return props.navigation?.isFocused()
```

Comment:
The app dies if we go back from the claims screen :laughing: ?

Reply:
The app does *not* die when we press back while claims screen is focused :D 
otherwise, we let it get processed (which just means actually going "back" within the app)